en/V3-Software_Platform_Requirements.md

```diff
@@ -30,9 +30,9 @@ The Linux operating system is one of the most popular in IoT. It has many featur
 | --  | ---------------------- | - | - | - |
 | **3.2.1** | Verify that the embedded operating system is configured according to industry best practices, benchmarks, and uses secure defaults. | ✓ | ✓ | ✓ |
 | **3.2.2** | Verify that all network services exposed by the device on every network interface are necessary services and unnecessary services are removed or disabled. | ✓ | ✓ | ✓ |
-| **3.2.3** | Verify that the device does not make use of of legacy or insecure protocols such as Telnet and FTP. | ✓ | ✓ | ✓ |
+| **3.2.3** | Verify that the device does not make use of of legacy or insecure protocols such as Telnet and FTP without compensating controls. | ✓ | ✓ | ✓ |
```

Comment:
Are there specific compensating controls that you have in mind? 
"without compensating controls" is something that can be appended to other requirements as well, and I believe this is implicit. Ideally you don't want to use insecure protocols. If you do choose to do so, then you accept the risk. If you have a risk, you can lower the potential impact or likelihood by introducing compensating / mitigating controls. 

Perhaps this is something to add to the introduction chapters.

Reply:
compensating controls might include data sanitization, application level authentication (instead of protocol level in SFTP) etc.

Reply:
I believe this is something that applies to every requirement. I propose to pick this up in the "Using the ISVS" chapter. See issue #63.